pkg/log/logshipper/logshipper.go

```diff
@@ -0,0 +1,127 @@
+package logshipper
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/agent/types"
+	"github.com/kolide/launcher/pkg/sendbuffer"
+)
+
+// TODO: consolidate this var, but where?
```

Comment:
Agreed but I don't have a good idea either 😁 maybe it could fit somewhere in pkg/agent?

Reply:
put it in pkg/agent/storage/keys.go